website/templates/project/registrations.mako

```diff
@@ -22,23 +32,157 @@
+        </div>
+        <div class="col-md-9">
+          <div>
+            % if 'admin' in user['permissions'] and not disk_saving_mode:
+            <a data-bind="css: {disabled: loading}" id="registerNode" class="btn btn-default" type="button">
+              <i class="fa fa-plus"></i>
+              New Draft Registration
+            </a>
+            % else:
+              Only project administrators can create, edit, or submit new draft registrations.
+            % endif
+          </div>
+          <br />
+          <div class="scripted" data-bind="foreach: drafts">
+            <li class="project list-group-item list-group-item-node">
+              <h4 data-bind="text: schema().title" ></h4>
+              <h4 class="list-group-item-heading">
+                <div class="progress progress-bar-md">
+                  <div class="progress-bar" role="progressbar" aria-valuemin="0" aria-valuemax="100"
+                       data-bind="attr.aria-completion: completion,
+                                  style: {width: completion() + '%'}">
+                    <span class="sr-only"></span>
+                  </div>
+                </div>
+                <small>
+                <p>initiated by: <span data-bind="text: initiator.fullname"></span>
+                <p>started: <span data-bind="text: initiated"></span></p>
+                <p>last updated: <span data-bind="text: updated"></span></p>
+                <span data-bind="if: requiresApproval">
+                    <div data-bind="if: isApproved">
+                        <div class="draft-status-badge bg-success"> Approved</div>
+                    </div>
+                    <div data-bind="ifnot: isApproved">
+                        <div class="draft-status-badge bg-warning"> Pending Approval </div>
+                    </div>
+                    <div data-bind="if: isPendingReview">
+                        <div class="draft-status-badge bg-warning"> Pending Review</div>
+                    </div>
+                </span>
+                </small>
+                <div class="row">
+                  <div class="col-md-10">
+                    <a class="btn btn-info"
+                       data-bind="click: $root.maybeWarn"><i style="margin-right: 5px;" class="fa fa-pencil"></i>Edit</a>
+                    <button class="btn btn-danger"
+                            data-bind="click: $root.deleteDraft"><i style="margin-right: 5px;" class="fa fa-times"></i>Delete</button>
+                  </div>
+                  <div class="col-md-1">
+                     <a class="btn btn-success" data-bind="attr.href: urls.register_page,
+                                                           tooltip: {
+                                                             placement: 'top',
+                                                             title: isApproved ? 'Finialize this draft' : 'This draft must be approved before it can be registered'
```

Comment:
Typo: should be "finalize".

(well, [probably](http://dictionary.reference.com/browse/finial?s=t) a typo, anyway!)